examples/autopost/auto_post_random.py

```
@@ -0,0 +1,40 @@
+import time
+import sys
+import os
+import yaml
+import glob
+from tqdm import tqdm
+
+sys.path.append(os.path.join(sys.path[0], '../../'))
+from instabot import Bot
+posted_pic_list = []
+try:
+    with open('random_pics.txt', 'r') as f:
+        posted_pic_list = f.read().splitlines()
+except:
```

don't user bare except
```except Exception```